frontends/web/src/routes/device/bitbox02/passphrase.tsx

```diff
@@ -35,345 +36,360 @@ const INFO_STEPS_DISABLE = 0;
 
 const CONTENT_MIN_HEIGHT = '38em';
 
-interface MnemonicPassphraseButtonProps {
+type TProps = {
     deviceID: string;
 }
 
-interface State {
-    infoStep: number;
-    passphraseEnabled?: boolean;
-    status: 'info' | 'progress' | 'success';
-    understood: boolean;
-}
+type TStatus = 'info' | 'progress' | 'success';
+type TPassphraseEnabled = boolean | undefined;
 
-type Props = MnemonicPassphraseButtonProps & TranslateProps;
+export const Passphrase = ({ deviceID }: TProps) => {
+  const { t } = useTranslation();
+  const navigate = useNavigate();
+  const [infoStep, setInfoStep] = useState(0);
+  const [status, setStatus] = useState<TStatus>('info');
+  // Can this be a useRef() ?
+  const [passphraseEnabled, setPassphraseEnabled] = useState<TPassphraseEnabled>();
+  const [understood, setUnderstood] = useState(false);
```

Comment:
understood state is only use within EnableInfo compnent, move the state there please.

Reply:
(understood along with other state that is only relevant to the enable wizard was moved into the component in the following commit. in general the following commit simplifies the entire code this way)

Reply:
actually since the refactor/improvement will be a separate PR I'll change this in this commit too